README.md

```diff
@@ -108,6 +109,13 @@ And additional requirements are in [**requirements.txt**](https://github.com/gar
   * Create admin `$ python manage.py createsuperuser`
   * Run the server `$ python manage.py runserver`
 
+
+ 
```

Comment:
Please remove these extra newlines, both here and below the Pro Tip section. Keep a single newline between sections.

Reply:
okay, did it.